tests/Unit/CssInlinerTest.php

```diff
@@ -1614,16 +1616,16 @@ public function orderedRulesAndSurroundingCssDataProvider()
                     // test all possible CSS before once
                     if (
                         ($cssBetween === ''
-                        && $cssAfter === '')
+                            && $cssAfter === '')
```

Comment:
This is slightly more readable that with the previous change in #767 :)

Reply:
;-)